packages/create-web-scripts-library/README.md

```diff
@@ -0,0 +1,35 @@
+# `create-web-scripts-library`
+
+Scaffold a [@spotify/web-scripts](https://github.com/spotify/web-scripts) library quickly. Code was directly inspired by [create-next-app](https://github.com/zeit/create-next-app).
+
+## Usage
+
+### With `yarn create`
+
+```sh
+yarn create @spotify/web-scripts-library my-cool-library
+```
+
+### With `npx`
+
+```sh
+npx @spotify/web-scripts-create-library my-cool-library
+```
+
+### Installed globally
+
+```sh
+npm i -g @spotify/web-scripts-create-library
+web-scripts-create-library my-cool-library
```

Comment:
I'd prefer if we avoided promoting a global install, they tend to lead to nothing but pain!

Reply:
it's a fact. happy to remove it, people can figure it out on their own if they really need it